src/NServiceBus.Transport.RabbitMQ/Connection/ChannelPool.cs

```diff
@@ -4,19 +4,19 @@ namespace NServiceBus.Transport.RabbitMQ
     using System.Collections.Concurrent;
     using global::RabbitMQ.Client;
 
-    sealed class ChannelProvider : IDisposable
+    sealed class ChannelPool : IDisposable
```

Comment:
The channels retrieved from this class are only for sending messages to the broker via the `BasicPublish` API call. These changes lose that distinction.

I think we need to retain that in some way.